shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -450,6 +450,14 @@ public FlutterActivity() {
     this.delegate = delegate;
   }
 
+  /**
+   * Returns the Android App Component exclusively attached to {@link
+   * io.flutter.embedding.engine.FlutterEngine}.
+   */
+  public ExclusiveAppComponent<Activity> getExclusiveAppComponent() {
```

Comment:
nit: this should be defined in the interface. `FlutterActivityAndFragmentDelegate.Host`

Reply:
Done.